code/modules/shuttle/navigation_computer.dm

```diff
@@ -327,7 +330,18 @@
 		if(console.z_lock.len && !(S.z in console.z_lock))
 			continue
 		if(console.jumpto_ports[S.id])
-			L[S.name] = S
+			L["([L.len])[S.name]"] = S
+
+	for(var/V in SSshuttle.beacons)
+		if(!V)
+			continue
```

Comment:
A null entry in a list like this is indicative of greater problems. Either take out the check or stack_trace() to make an error